cfgov/unprocessed/apps/youth-employment-success/js/budget-form-view.js

```diff
@@ -47,30 +42,57 @@ function BudgetFormView( element, { store } ) {
   const _handleEarnedInput = _handleInput( updateEarnedAction );
   const _handleSpentInput = _handleInput( updateSpentAction );
 
+  /**
+   * Update the input element with formatted value, dispatch that value to the store
+   * @param {Object} updateObj Object with updated data from form control event dispatch
+   * @param {Object} updateObj.event The raw DOM event
+   * @param {Object} updateObj.value The value of the form control element
+   */
+  function _handleEarnedBlur( { event, value } ) {
+    event.target.value = toPrecision( value, 2 );
+    _handleEarnedInput( { event, value: event.target.value } );
+  }
+
+  /**
+   * Update the input element with formatted value, dispatch that value to the store
+   * @param {Object} updateObj Object with updated data from form control event dispatch
+   * @param {Object} updateObj.event The raw DOM event
+   * @param {Object} updateObj.value The value of the form control element
+   */
+  function _handleSpentBlur( { event, value } ) {
+    event.target.value = value ? toPrecision( value, 2 ) : '';
+    _handleSpentInput( { event, value: event.target.value } );
+  }
+
   const _moneyEarnedEl = inputView(
     _dom.querySelector( `.${ CLASSES.EARNED_INPUT }` ),
     {
       events: {
-        input: _handleEarnedInput
+        input: _handleEarnedInput,
+        blur: _handleEarnedBlur
       }
     }
   );
   const _moneySpentEl = inputView(
     _dom.querySelector( `.${ CLASSES.SPENT_INPUT }` ),
     {
       events: {
-        input: _handleSpentInput
+        input: _handleSpentInput,
+        blur: _handleSpentBlur
       }
     }
   );
   const _moneyRemainingEl = _dom.querySelector( `.${ CLASSES.REMAINING }` );
 
   /**
    * Update dom node with amount remaining in user's budget
+   * @param {Object} _ The previous state of the store
```

Comment:
Ah sorry, so the `_` is the convention I was taught for when a variable is passed to a function but is unused. I've had linters complain about this.

Reply:
Does this make sense? It's possible our linters aren't configured this way and an unused variable in the function declaration is fine, but I feel like its complained at me before.

Reply:
Ahh, got it! (Again) 
So I bet the reason for this is https://blog.adamzolo.com/eslint-ignore-parameters-with-underscore/, whereby instead of individually ignoring linter complaints of unused variables, the underscore is used as a blanket convention to get around the linter. I wonder why a leading underscore regex couldn't be used though, like `_previousState`? Anyway, this is fine, but let me know if you'd like input on how our linter should handle this.

Reply:
Hmm interesting. So to me, coming from ruby, the `_` is the convention (or it was when I last used it) to indicate an unused argument. I wonder if that is why this rule exists in `eslint`?

I like the `_previousState` idea, but it's also the same the way we already declare private variables, and that has the potential to be confusing as well.

This kind of ambiguity is why I generally prefer to pass a single object as a function argument, and let the function take the data it needs.